flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/IntervalJoinRestoreTest.java

```diff
@@ -35,7 +35,13 @@ public IntervalJoinRestoreTest() {
     public List<TableTestProgram> programs() {
         return Arrays.asList(
                 IntervalJoinTestPrograms.INTERVAL_JOIN_EVENT_TIME,
+                IntervalJoinTestPrograms.INTERVAL_JOIN_EVENT_TIME_UPDATING_SOURCE,
```

Comment:
We don't need a restore test. Restoring doesn't need to be tested. 
I added a `SemanticTestBase` recently to test an execution only:
Take this as an example: https://github.com/apache/flink/pull/26236